Plugin don´t work in Gitbucket 4.20.0

Hi,
when I try to load the plugin in Gitbucket 4.20.0 I only get the following error message. 
Does anyone know what I might have done wrong?


```
16:00:06.241 [qtp250421012-12] ERROR gitbucket.core.plugin.PluginRegistry - Error during plugin initialization: gitbucket-fess-plugin_2.12-1.2.0.jar
java.lang.reflect.InvocationTargetException: null
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
	at gitbucket.core.plugin.PluginRegistry$.$anonfun$initialize$3(PluginRegistry.scala:275)
	at gitbucket.core.plugin.PluginRegistry$.$anonfun$initialize$3$adapted(PluginRegistry.scala:267)
	at gitbucket.core.plugin.PluginRegistry$$$Lambda$286/790538253.apply(Unknown Source)
	at scala.collection.immutable.List.foreach(List.scala:389)
	at gitbucket.core.plugin.PluginRegistry$.initialize(PluginRegistry.scala:267)
	at gitbucket.core.plugin.PluginRegistry$.reload(PluginRegistry.scala:209)
	at gitbucket.core.controller.SystemSettingsControllerBase.$anonfun$$init$$15(SystemSettingsController.scala:222)
	at gitbucket.core.controller.SystemSettingsControllerBase$$Lambda$505/1100288091.apply(Unknown Source)
	at org.scalatra.ScalatraBase.liftAction(ScalatraBase.scala:280)
	at org.scalatra.ScalatraBase.$anonfun$invoke$1(ScalatraBase.scala:274)
	at org.scalatra.ScalatraBase$$Lambda$1536/1670691471.apply(Unknown Source)
	at org.scalatra.ApiFormats.withRouteMultiParams(ApiFormats.scala:178)
	at org.scalatra.ApiFormats.withRouteMultiParams$(ApiFormats.scala:164)
	at gitbucket.core.controller.ControllerBase.withRouteMultiParams(ControllerBase.scala:33)
	at org.scalatra.ScalatraBase.invoke(ScalatraBase.scala:273)
	at org.scalatra.ScalatraBase.invoke$(ScalatraBase.scala:272)
	at gitbucket.core.controller.ControllerBase.org$scalatra$json$JsonSupport$$super$invoke(ControllerBase.scala:33)
	at org.scalatra.json.JsonSupport.$anonfun$invoke$1(JsonSupport.scala:88)
	at org.scalatra.json.JsonSupport$$Lambda$1516/1869009909.apply(Unknown Source)
	at org.scalatra.ApiFormats.withRouteMultiParams(ApiFormats.scala:178)
	at org.scalatra.ApiFormats.withRouteMultiParams$(ApiFormats.scala:164)
	at gitbucket.core.controller.ControllerBase.withRouteMultiParams(ControllerBase.scala:33)
	at org.scalatra.json.JsonSupport.invoke(JsonSupport.scala:82)
	at org.scalatra.json.JsonSupport.invoke$(JsonSupport.scala:81)
	at gitbucket.core.controller.ControllerBase.invoke(ControllerBase.scala:33)
	at org.scalatra.ScalatraBase.$anonfun$runRoutes$3(ScalatraBase.scala:248)
	at org.scalatra.ScalatraBase$$Lambda$1633/625755046.apply(Unknown Source)
	at scala.Option.flatMap(Option.scala:171)
	at org.scalatra.ScalatraBase.$anonfun$runRoutes$1(ScalatraBase.scala:246)
	at org.scalatra.ScalatraBase$$Lambda$1579/1934518258.apply(Unknown Source)
	at scala.collection.immutable.Stream.flatMap(Stream.scala:490)
	at org.scalatra.ScalatraBase.runRoutes(ScalatraBase.scala:245)
	at org.scalatra.ScalatraBase.runRoutes$(ScalatraBase.scala:243)
	at gitbucket.core.controller.ControllerBase.runRoutes(ControllerBase.scala:33)
	at org.scalatra.ScalatraBase.runActions$1(ScalatraBase.scala:169)
	at org.scalatra.ScalatraBase.$anonfun$executeRoutes$6(ScalatraBase.scala:181)
	at org.scalatra.ScalatraBase$$Lambda$1496/1254715465.apply$mcV$sp(Unknown Source)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at org.scalatra.ScalatraBase.cradleHalt(ScalatraBase.scala:199)
	at org.scalatra.ScalatraBase.executeRoutes(ScalatraBase.scala:181)
	at org.scalatra.ScalatraBase.executeRoutes$(ScalatraBase.scala:150)
	at gitbucket.core.controller.ControllerBase.executeRoutes(ControllerBase.scala:33)
	at org.scalatra.ScalatraBase.$anonfun$handle$1(ScalatraBase.scala:123)
	at org.scalatra.ScalatraBase$$Lambda$1494/1272469045.apply$mcV$sp(Unknown Source)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.DynamicVariable.withValue(DynamicVariable.scala:58)
	at org.scalatra.DynamicScope.withResponse(DynamicScope.scala:79)
	at org.scalatra.DynamicScope.withResponse$(DynamicScope.scala:77)
	at gitbucket.core.controller.ControllerBase.withResponse(ControllerBase.scala:33)
	at org.scalatra.DynamicScope.$anonfun$withRequestResponse$1(DynamicScope.scala:59)
	at org.scalatra.DynamicScope$$Lambda$1495/1088739156.apply(Unknown Source)
	at scala.util.DynamicVariable.withValue(DynamicVariable.scala:58)
	at org.scalatra.DynamicScope.withRequest(DynamicScope.scala:70)
	at org.scalatra.DynamicScope.withRequest$(DynamicScope.scala:68)
	at gitbucket.core.controller.ControllerBase.withRequest(ControllerBase.scala:33)
	at org.scalatra.DynamicScope.withRequestResponse(DynamicScope.scala:58)
	at org.scalatra.DynamicScope.withRequestResponse$(DynamicScope.scala:56)
	at gitbucket.core.controller.ControllerBase.withRequestResponse(ControllerBase.scala:33)
	at org.scalatra.ScalatraBase.handle(ScalatraBase.scala:123)
	at org.scalatra.ScalatraBase.handle$(ScalatraBase.scala:119)
	at gitbucket.core.controller.ControllerBase.org$scalatra$servlet$ServletBase$$super$handle(ControllerBase.scala:33)
	at org.scalatra.servlet.ServletBase.handle(ServletBase.scala:53)
	at org.scalatra.servlet.ServletBase.handle$(ServletBase.scala:46)
	at gitbucket.core.controller.ControllerBase.org$scalatra$FlashMapSupport$$super$handle(ControllerBase.scala:33)
	at org.scalatra.FlashMapSupport.$anonfun$handle$1(FlashMap.scala:170)
	at org.scalatra.FlashMapSupport$$Lambda$1487/1354626090.apply$mcV$sp(Unknown Source)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.DynamicVariable.withValue(DynamicVariable.scala:58)
	at org.scalatra.DynamicScope.withRequest(DynamicScope.scala:70)
	at org.scalatra.DynamicScope.withRequest$(DynamicScope.scala:68)
	at gitbucket.core.controller.ControllerBase.withRequest(ControllerBase.scala:33)
	at org.scalatra.FlashMapSupport.handle(FlashMap.scala:145)
	at org.scalatra.FlashMapSupport.handle$(FlashMap.scala:144)
	at gitbucket.core.controller.ControllerBase.handle(ControllerBase.scala:33)
	at org.scalatra.ScalatraFilter.$anonfun$doFilter$1(ScalatraFilter.scala:41)
	at org.scalatra.ScalatraFilter$$Lambda$1486/1575987880.apply$mcV$sp(Unknown Source)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.DynamicVariable.withValue(DynamicVariable.scala:58)
	at org.scalatra.ScalatraFilter.doFilter(ScalatraFilter.scala:41)
	at org.scalatra.ScalatraFilter.doFilter$(ScalatraFilter.scala:36)
	at gitbucket.core.controller.ControllerBase.doFilter(ControllerBase.scala:72)
	at gitbucket.core.servlet.CompositeScalatraFilter.$anonfun$doFilter$2(CompositeScalatraFilter.scala:48)
	at gitbucket.core.servlet.CompositeScalatraFilter.$anonfun$doFilter$2$adapted(CompositeScalatraFilter.scala:46)
	at gitbucket.core.servlet.CompositeScalatraFilter$$Lambda$1635/1832397776.apply(Unknown Source)
	at scala.collection.immutable.List.foreach(List.scala:389)
	at scala.collection.generic.TraversableForwarder.foreach(TraversableForwarder.scala:35)
	at scala.collection.generic.TraversableForwarder.foreach$(TraversableForwarder.scala:35)
	at scala.collection.mutable.ListBuffer.foreach(ListBuffer.scala:44)
	at gitbucket.core.servlet.CompositeScalatraFilter.doFilter(CompositeScalatraFilter.scala:46)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1637)
	at gitbucket.core.servlet.PluginControllerFilter.doFilter(PluginControllerFilter.scala:44)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1637)
	at org.scalatra.ScalatraFilter.$anonfun$$init$$1(ScalatraFilter.scala:81)
	at org.scalatra.ScalatraFilter.$anonfun$$init$$1$adapted(ScalatraFilter.scala:81)
	at org.scalatra.ScalatraFilter$$Lambda$307/1109461985.apply(Unknown Source)
	at org.scalatra.ScalatraBase.$anonfun$matchOtherMethods$1(ScalatraBase.scala:324)
	at org.scalatra.ScalatraBase$$Lambda$1592/345326531.apply(Unknown Source)
	at org.scalatra.ScalatraBase.liftAction(ScalatraBase.scala:280)
	at org.scalatra.ScalatraBase.matchOtherMethods(ScalatraBase.scala:324)
	at org.scalatra.ScalatraBase.$anonfun$executeRoutes$4(ScalatraBase.scala:172)
	at org.scalatra.ScalatraBase$$Lambda$1582/1389893181.apply(Unknown Source)
	at scala.Option.orElse(Option.scala:289)
	at org.scalatra.ScalatraBase.$anonfun$executeRoutes$3(ScalatraBase.scala:172)
	at org.scalatra.ScalatraBase$$Lambda$1581/498252762.apply(Unknown Source)
	at scala.Option.getOrElse(Option.scala:121)
	at org.scalatra.ScalatraBase.runActions$1(ScalatraBase.scala:172)
	at org.scalatra.ScalatraBase.$anonfun$executeRoutes$6(ScalatraBase.scala:181)
	at org.scalatra.ScalatraBase$$Lambda$1496/1254715465.apply$mcV$sp(Unknown Source)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at org.scalatra.ScalatraBase.cradleHalt(ScalatraBase.scala:199)
	at org.scalatra.ScalatraBase.executeRoutes(ScalatraBase.scala:181)
	at org.scalatra.ScalatraBase.executeRoutes$(ScalatraBase.scala:150)
	at gitbucket.core.controller.ControllerBase.executeRoutes(ControllerBase.scala:33)
	at org.scalatra.ScalatraBase.$anonfun$handle$1(ScalatraBase.scala:123)
	at org.scalatra.ScalatraBase$$Lambda$1494/1272469045.apply$mcV$sp(Unknown Source)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.DynamicVariable.withValue(DynamicVariable.scala:58)
	at org.scalatra.DynamicScope.withResponse(DynamicScope.scala:79)
	at org.scalatra.DynamicScope.withResponse$(DynamicScope.scala:77)
	at gitbucket.core.controller.ControllerBase.withResponse(ControllerBase.scala:33)
	at org.scalatra.DynamicScope.$anonfun$withRequestResponse$1(DynamicScope.scala:59)
	at org.scalatra.DynamicScope$$Lambda$1495/1088739156.apply(Unknown Source)
	at scala.util.DynamicVariable.withValue(DynamicVariable.scala:58)
	at org.scalatra.DynamicScope.withRequest(DynamicScope.scala:70)
	at org.scalatra.DynamicScope.withRequest$(DynamicScope.scala:68)
	at gitbucket.core.controller.ControllerBase.withRequest(ControllerBase.scala:33)
	at org.scalatra.DynamicScope.withRequestResponse(DynamicScope.scala:58)
	at org.scalatra.DynamicScope.withRequestResponse$(DynamicScope.scala:56)
	at gitbucket.core.controller.ControllerBase.withRequestResponse(ControllerBase.scala:33)
	at org.scalatra.ScalatraBase.handle(ScalatraBase.scala:123)
	at org.scalatra.ScalatraBase.handle$(ScalatraBase.scala:119)
	at gitbucket.core.controller.ControllerBase.org$scalatra$servlet$ServletBase$$super$handle(ControllerBase.scala:33)
	at org.scalatra.servlet.ServletBase.handle(ServletBase.scala:53)
	at org.scalatra.servlet.ServletBase.handle$(ServletBase.scala:46)
	at gitbucket.core.controller.ControllerBase.org$scalatra$FlashMapSupport$$super$handle(ControllerBase.scala:33)
	at org.scalatra.FlashMapSupport.$anonfun$handle$1(FlashMap.scala:170)
	at org.scalatra.FlashMapSupport$$Lambda$1487/1354626090.apply$mcV$sp(Unknown Source)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.DynamicVariable.withValue(DynamicVariable.scala:58)
	at org.scalatra.DynamicScope.withRequest(DynamicScope.scala:70)
	at org.scalatra.DynamicScope.withRequest$(DynamicScope.scala:68)
	at gitbucket.core.controller.ControllerBase.withRequest(ControllerBase.scala:33)
	at org.scalatra.FlashMapSupport.handle(FlashMap.scala:145)
	at org.scalatra.FlashMapSupport.handle$(FlashMap.scala:144)
	at gitbucket.core.controller.ControllerBase.handle(ControllerBase.scala:33)
	at org.scalatra.ScalatraFilter.$anonfun$doFilter$1(ScalatraFilter.scala:41)
	at org.scalatra.ScalatraFilter$$Lambda$1486/1575987880.apply$mcV$sp(Unknown Source)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.util.DynamicVariable.withValue(DynamicVariable.scala:58)
	at org.scalatra.ScalatraFilter.doFilter(ScalatraFilter.scala:41)
	at org.scalatra.ScalatraFilter.doFilter$(ScalatraFilter.scala:36)
	at gitbucket.core.controller.ControllerBase.doFilter(ControllerBase.scala:72)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1637)
	at gitbucket.core.servlet.TransactionFilter.$anonfun$doFilter$1(TransactionFilter.scala:39)
	at gitbucket.core.servlet.TransactionFilter.$anonfun$doFilter$1$adapted(TransactionFilter.scala:30)
	at gitbucket.core.servlet.TransactionFilter$$Lambda$1482/214383121.apply(Unknown Source)
	at com.github.takezoe.slick.blocking.BlockingJdbcProfile$BlockingAPI$BlockingDatabase.$anonfun$withTransaction$2(BlockingProfile.scala:207)
	at com.github.takezoe.slick.blocking.BlockingJdbcProfile$BlockingAPI$BlockingDatabase$$Lambda$195/1028145537.apply(Unknown Source)
	at slick.JdbcProfileBlockingSession$BlockingSession.withTransaction(TransactionalJdbcBackend.scala:26)
	at com.github.takezoe.slick.blocking.BlockingJdbcProfile$BlockingAPI$BlockingDatabase.$anonfun$withTransaction$1(BlockingProfile.scala:207)
	at com.github.takezoe.slick.blocking.BlockingJdbcProfile$BlockingAPI$BlockingDatabase$$Lambda$194/1636588948.apply(Unknown Source)
	at com.github.takezoe.slick.blocking.BlockingJdbcProfile$BlockingAPI$BlockingDatabase.withSession(BlockingProfile.scala:200)
	at com.github.takezoe.slick.blocking.BlockingJdbcProfile$BlockingAPI$BlockingDatabase.withTransaction(BlockingProfile.scala:207)
	at gitbucket.core.servlet.TransactionFilter.doFilter(TransactionFilter.scala:30)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1629)
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:533)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:143)
	at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:548)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:190)
	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:1595)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:188)
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1253)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:168)
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:473)
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1564)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:166)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1155)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)
	at org.eclipse.jetty.server.handler.StatisticsHandler.handle(StatisticsHandler.java:169)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.eclipse.jetty.server.Server.handle(Server.java:561)
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:334)
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:251)
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:279)
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:104)
	at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:124)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:679)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:597)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.NoClassDefFoundError: io/github/gitbucket/scalatra/forms/package$ValueType
	at Plugin.<init>(Plugin.scala:22)
	... 194 common frames omitted
Caused by: java.lang.ClassNotFoundException: io.github.gitbucket.scalatra.forms.package$ValueType
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 195 common frames omitted
```

--- a/build.sbt
+++ b/build.sbt
@@ -4,20 +4,9 @@ organization := "org.codelibs.gitbucket"
 
 version := "1.2.1-SNAPSHOT"
 
-scalaVersion := "2.12.1"
+scalaVersion := "2.12.4"
 
-lazy val root = (project in file(".")).enablePlugins(SbtTwirl)
-
-resolvers ++= Seq(
-  Resolver.jcenterRepo
-)
-
-libraryDependencies ++= Seq(
-  "io.github.gitbucket" %% "gitbucket"        % "4.15.0" % "provided",
-  "com.typesafe.play"   %% "twirl-compiler"   % "1.3.0" % "provided",
-  "org.json4s"          %% "json4s-jackson"   % "3.5.0",
-  "javax.servlet"       % "javax.servlet-api" % "3.0.1" % "provided"
-)
+gitbucketVersion := "4.20.0"
 
 publishMavenStyle := true
 

--- a/project/build.properties
+++ b/project/build.properties
@@ -1,1 +1,1 @@
-sbt.version = 0.13.13
+sbt.version = 1.1.0

--- a/project/plugins.sbt
+++ b/project/plugins.sbt
@@ -1,2 +1,2 @@
-addSbtPlugin("com.typesafe.sbt" % "sbt-twirl" % "1.3.0")
-addSbtPlugin("com.geirsson" % "sbt-scalafmt" % "0.5.6")
+addSbtPlugin("io.github.gitbucket" % "sbt-gitbucket-plugin" % "1.2.0")
+addSbtPlugin("com.geirsson" % "sbt-scalafmt" % "1.4.0")

--- a/src/main/scala/org/codelibs/gitbucket/fess/controller/FessSearchController.scala
+++ b/src/main/scala/org/codelibs/gitbucket/fess/controller/FessSearchController.scala
@@ -7,9 +7,6 @@ import org.codelibs.gitbucket.fess.html
 import gitbucket.core.util._
 import gitbucket.core.util.Implicits._
 
-import scala.reflect.ClassTag
-import scala.reflect.runtime.universe._
-
 class FessSearchController
     extends FessSearchControllerBase
     with FessSearchService

--- a/src/main/scala/org/codelibs/gitbucket/fess/controller/FessSettingsController.scala
+++ b/src/main/scala/org/codelibs/gitbucket/fess/controller/FessSettingsController.scala
@@ -2,8 +2,8 @@ package org.codelibs.gitbucket.fess.controller
 
 import gitbucket.core.controller.ControllerBase
 import gitbucket.core.util.AdminAuthenticator
+import org.scalatra.forms._
 import org.codelibs.gitbucket.fess.html
-import io.github.gitbucket.scalatra.forms._
 import org.codelibs.gitbucket.fess.service.FessSettingsService
 import org.codelibs.gitbucket.fess.service.FessSettingsService._
 

